bridge/core/rust_api/document.cc

```diff
@@ -29,6 +29,82 @@ RustValue<Element, ElementRustMethods> DocumentRustMethods::CreateElement(
   return {.value = new_element, .method_pointer = To<ElementRustMethods>(new_element->rustMethodPointer())};
 }
 
+RustValue<Element, ElementRustMethods> DocumentRustMethods::CreateElementWithOptions(
```

Comment:
It seems like they are no DOM API called `createElementWithOptions`.

Reply:
The API design should be followed the WhatWG DOM spec, especially mostly the same as the `web-sys` crate.

https://crates.io/crates/web-sys

So the top level rust web framework can easy migrate their dependencies from 'web-sys` to `webf-sys`.

Reply:
Great. That's just what I need~